code/modules/supermatter/supermatter.dm

```diff
@@ -483,6 +488,11 @@
 	..()
 	set_frequency(frequency)
 
+/obj/machinery/computer/supermatter/process()
```

Comment:
does this even get called? I don't see this being added to processing_list. what was wrong with the clearly superior on_destroyed approach I suggested?

Reply:
Yeah it gets called, I did testing. I am pretty sure all machinery gets added to processing objects for power related purposes. Your method was fine too, but I thought this was more legible.

Reply:
Yeah all machines are automatically processing.

It actually checks if `process()` is overriden (does not return `PROCESS_KILL`) and culls types without process overriden.